app/com/linkedin/drelephant/ElephantRunner.java

```diff
@@ -46,7 +46,7 @@
   private static final Logger logger = Logger.getLogger(ElephantRunner.class);
 
   private static final long WAIT_INTERVAL = 60 * 1000;      // Interval between fetches and retries
-  private static final int EXECUTOR_NUM = 3;                // The number of executor threads to analyse the jobs
+  private static final int EXECUTOR_NUM = 5;                // The number of executor threads to analyse the jobs
```

Comment:
This was decreased to 3 when sampling was added since after sampling each executor would have taken less time to execute and the queue won't be overwhelmed. We need to change it back to 5 because now each executor may take more time and the queue size will increase if we have only 3 executors.